src/crypting/Main.java

```diff
@@ -0,0 +1,40 @@
+package crypting;
+
+import java.io.IOException;
+
+public class Main {
+
+    public static void main(String[] args) {
+        CryptConfiguration config;
+        config = new CryptConfiguration(new CliParser(args));
+        Cryptor cryptor = new Cryptor(config.getCipher());
+
+        try {
+            String inputData = getInputData(config);
+            String cryptedData = getCryptedData(config, cryptor, inputData);
```

Comment:
Small changes, please rename it to `cryptData`

Reply:
Done